doc/EXAMPLES.rst

```diff
@@ -0,0 +1,69 @@
+############################################
+             TeleSculptor Examples
+############################################
+
+We provide two suggested video clips and a description of how to run a
+project in the TeleSculptor GUI. For a more thorough description of
+TeleSculptor workflows, please see the
+`User Guide <../doc/TeleSculptor-v1.1-User-Guide.pdf>`_.
+
+Data
+====
+We recommend that new users try one of the following videos released as part
+of the `VIRAT Video Dataset`_.
+You will need to accept the Terms of Use before you proceed to the download.
+
+
+09172008flight1tape3_2.mpg_: The aircraft makes about one complete orbit over
+a site in Fort A.P. Hill, Virginia. The stare point remains fixed on the center
+of the scene and the field of view is quite narrow. The scene is composed
+primarily of buildings and vehicles.
+
+09152008flight2tape2_4.mpg_: This is video of the same region but taken in a
+different style. The field of view is much larger and the stare point moves
+substantially. In addition to the buildings and vehicles which are the focus of
+the other clip, this video includes wide shots of roads and vegetation.
+
+If you are considering running TeleSculptor on different data, there are several things to
+keep in mind. The first is that TeleSculptor is optimized for aerial data and
+results may vary on data collected in other ways.
+Additionally, the way to data is captured has a significant impact on the
+quality of the model. It is important to have multiple views of the same regions
+from different directions and if possible the camera should complete a full rotation about the scene.
+It is best not to change the zoom of your lens and if you do it should be done
+slowly and gradually. Finally, you should avoid switching between different
+modalities, for example recording part of your video in color and the rest in infrared.
+If you are capturing images rather than video, it is important that there is
+substantial overlap in the content seen between different images.
+
+Setup
```

Comment:
These next three sections are redundant with user guide.  This is all good stuff, but I'm sure we will forget to maintain this text as TeleSculptor is updated.  It's probably better to have this information in a common spot, which is the user guide you've already linked to above.  I would remove the sections below.

In the future we should probably make the user guide something other than a PDF (generated from a Word Doc), but that's a topic for another day.

Reply:
I should have read the user guide first. Could the `Data` section just become part of the README since it is small and isn't associated with any other files?

Reply:
Yes, I suppose it could. As written it is fairly brief and doesn't provide additional files.  However, I think over time I'd like to add more files here.  This would include a still image from each video and maybe a resulting mesh file.  We may add other video sources later too.  So maybe it's best just to keep it in it's own folder for now.